wc-admin.php

```diff
@@ -43,8 +43,16 @@ function wc_admin_plugins_notice() {
  * @return bool
  */
 function dependencies_satisfied() {
-	return ( defined( 'GUTENBERG_DEVELOPMENT_MODE' ) || defined( 'GUTENBERG_VERSION' ) )
-			&& class_exists( 'WooCommerce' ) && version_compare( WC_VERSION, '3.5', '>' );
+	$woocommerce_minimum_met = class_exists( 'WooCommerce' ) && version_compare( WC_VERSION, '3.5', '>' );
+	if ( ! $woocommerce_minimum_met ) {
+		return false;
+	}
+
+	$wordpress_version            = get_bloginfo( 'version' );
+	$wordpress_includes_gutenberg = version_compare( $wordpress_version, '4.9.9', '>' );
```

Comment:
Since we have the known issue of `SCRIPT_DEBUG` needing to be on, should we add that as a required condition to this line? That would allow us to ensure `wc-admin` will function in 5.0 until the other issue is resolved - and we could cut a release too!

```
$wordpress_includes_gutenberg = version_compare( $wordpress_version, '4.9.9', '>' ) && is_defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG;
```

Reply:
Changed in https://github.com/woocommerce/wc-admin/pull/774/commits/ca754100cd004a12361eb7e05a3760cf6122a4ef